packages/ava/src/insight/algorithms/base/compare.ts

```diff
@@ -1,4 +1,10 @@
-export const ascending = (left: any, right: any) => {
+/**
+ * lodash sortby asc function
+ * @param left any
+ * @param right any
+ * @returns number
+ */
+export function ascending(left: any, right: any) {
```

Comment:
Should we modify all the `any` to `unknown`?

Reply:
done